### PR TITLE
Object storage cli commands

### PR DESCRIPTION
**Pull request by tvoinarovskyi**

### CODE REVIEW

**Comment by asvetlov** [comment]
fix linter please

File: neuromation/cli/blob_storage.py [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+        # Similar to `ls -1`, default for non-terminal on UNIX. We show full uris of
+        # blobs, thus column formatting does not work too well.
+        formatter = SimpleBlobFormatter(root.color)
+
+    if len(blob_listings) > 1:
+        buffer = []
+        for uri, listing in zip(uris, blob_listings):
+            buffer.append(click.style(str(uri), bold=True) + ":")
+            buffer.extend(formatter(listing))
+        pager_maybe("".join(buffer), root.tty, root.terminal_size)
```
Comment (serhiy-storchaka) [inline neuromation/cli/blob_storage.py]: The first argument of `pager_maybe()` is an iterable of lines.
```suggestion
        pager_maybe(buffer, root.tty, root.terminal_size)
```

File: neuromation/cli/blob_storage.py [hunk 2]
```diff
@@ -0,0 +1,368 @@
...
+    blob = root.client.blob_storage
+
+    blob_listings: List[List[BlobListings]] = []
+
+    if not uris:
+        # List Buckets instead of blobs in bucket
+        listing: List[BlobListings] = []
+        blob_listings.append(listing)
+        for bucket in await blob.list_buckets():
+            listing.append(bucket)
```
[the remark this answers is not in the document]
Comment (tvoinarovskyi) [reply neuromation/cli/blob_storage.py]: Mypy can't work out how it's a list of Union of 2 types and you want to put it inside of a List of Union of 3 Types... Maybe it makes sense to just force no type check here.

File: neuromation/cli/blob_storage.py [hunk 3]
```diff
@@ -0,0 +1,368 @@
...
+        for uri in uris:
+            bucket_name, key = blob._extract_bucket_and_key(uri)
+
+            listing = []
+            blob_listings.append(listing)
+
+            for res in await blob.list_blobs(
+                bucket_name=bucket_name, prefix=key, recursive=recursive,
+            ):
+                listing.append(res)
```
[the remark this answers is not in the document]
Comment (tvoinarovskyi) [reply neuromation/cli/blob_storage.py]: Same reason

**Comment by tvoinarovskyi** [comment]
@serhiy-storchaka @asvetlov Should be good for the last review! All e2e tests should be fixed including glob test.

**Pull request merged**